packages/lodestar-cli/src/cmds/dev/handler.ts

```diff
@@ -35,9 +36,9 @@ export async function devHandler(args: IDevArgs & IGlobalArgs): Promise<void> {
   beaconNodeOptions.set({network: {discv5: {enr}}});
 
   // Custom paths different than regular beacon, validator paths
-  const rootDir = path.join(args.rootDir || defaultRootDir, "dev");
-  const chainDir = path.join(rootDir, "beacon");
-  const validatorsDir = path.join(rootDir, "validators");
+  const globalPaths = getGlobalPaths({...args, testnet: "dev" as TestnetName});
```

Comment:
I don't think dev chain should be stored in home directory. Let's keep it in working directory